schedule/yast/textmode/ha_textmode.yaml

```diff
@@ -0,0 +1,32 @@
+name: ha_textmode
+description: |
+  Installation sles with ha in textmode which selects System Role 'Text Mode'.
+vars:
+  VIDEOMODE: text
```

Comment:
could we also move `VIDEOMODE` AND `DESKTOP` to job group yaml, I think this will help in the future.